pkg/kobject/kobject.go

```diff
@@ -54,31 +57,32 @@ type ConvertOptions struct {
-	Network       []string          `compose:"network",bundle:"Networks"`
-	Labels        map[string]string `compose:"labels",bundle:"Labels"`
-	Annotations   map[string]string `compose:"",bundle:""`
-	CPUSet        string            `compose:"cpuset",bundle:""`
-	CPUShares     int64             `compose:"cpu_shares",bundle:""`
-	CPUQuota      int64             `compose:"cpu_quota",bundle:""`
-	CapAdd        []string          `compose:"cap_add",bundle:""`
-	CapDrop       []string          `compose:"cap_drop",bundle:""`
-	Expose        []string          `compose:"expose",bundle:""`
-	Privileged    bool              `compose:"privileged",bundle:""`
-	Restart       string            `compose:"restart",bundle:""`
-	User          string            `compose:"user",bundle:"User"`
-	VolumesFrom   []string          `compose:"volumes_from",bundle:""`
-	ServiceType   string            `compose:"kompose.service.type",bundle:""`
-	Build         string            `compose:"build",bundle:""`
-	ExposeService string            `compose:"kompose.service.expose",bundle:""`
-	Stdin         bool              `compose:"stdin_open",bundle:""`
-	Tty           bool              `compose:"tty",bundle:""`
+	Image         string              `compose:"image",bundle:"Image"`
+	Environment   []EnvVar            `compose:"environment",bundle:"Env"`
+	Port          []Ports             `compose:"ports",bundle:"Ports"`
+	Command       []string            `compose:"command",bundle:"Command"`
+	WorkingDir    string              `compose:"",bundle:"WorkingDir"`
+	Args          []string            `compose:"args",bundle:"Args"`
+	Volumes       []string            `compose:"volumes",bundle:"Volumes"`
+	Network       []string            `compose:"network",bundle:"Networks"`
+	Labels        map[string]string   `compose:"labels",bundle:"Labels"`
+	Annotations   map[string]string   `compose:"",bundle:""`
+	CPUSet        string              `compose:"cpuset",bundle:""`
+	CPUShares     int64               `compose:"cpu_shares",bundle:""`
+	CPUQuota      int64               `compose:"cpu_quota",bundle:""`
+	CapAdd        []string            `compose:"cap_add",bundle:""`
+	CapDrop       []string            `compose:"cap_drop",bundle:""`
+	Expose        []string            `compose:"expose",bundle:""`
+	Privileged    bool                `compose:"privileged",bundle:""`
+	Restart       string              `compose:"restart",bundle:""`
+	User          string              `compose:"user",bundle:"User"`
+	VolumesFrom   []string            `compose:"volumes_from",bundle:""`
+	ServiceType   string              `compose:"kompose.service.type",bundle:""`
+	Build         string              `compose:"build",bundle:""`
+	ExposeService string              `compose:"kompose.service.expose",bundle:""`
+	Stdin         bool                `compose:"stdin_open",bundle:""`
+	Tty           bool                `compose:"tty",bundle:""`
+	MemLimit      yaml.MemStringorInt `compose:"mem_limit",bundle:""`
```

Comment:
I think it would be better to have MemLimit as int here and convert it to in loader. This is how it is done for CPUShare and CPUQuota, it would be nice to have it in same way for this.